### DISCUSSION TITLE
Add support matrix for demos and downstream images

### DISCUSSION THREAD

**Discussion by xfiderek** [space-ros/space-ros#229] [Ideas]
I'd like to propose the creation and maintenance of a support matrix in the README or other documentation. The goal is to provide a single, clear view of which images (e.g., nav2, moveit2) and which demos (e.g., space_robots) have been tested for a given release of SpaceROS.

This matrix would make it easier for users to select the most suitable or mature version for their specific use case, based on the testing coverage. It would also help the community quickly identify which combinations are fully supported and which may require further testing.

An example matrix could look like this:

|                    | humble-2024.04.0 | humble-2024.07.0 | jazzy-2024.10.0 |
|:------------------:|:----------------:|:----------------:|:---------------:|
| navigation2 docker |         ✅        |         ✅        |        ❓        |
|   moveit2 docker   |         ✅        |         ✅        |        ❓        |
|  space_robots demo |         ✅        |         ⚠️        |        ❓        |
|         ...        |        ...       |        ...       |       ...       |
  
With ✅  meaning tested and supported, ⚠️ meaning unstable/experimental, ❓ meaning untested.

Let me know your thoughts or suggestions on this idea!

**Comment by Bckempa** [comment]
I'm a big fan of matrices and this looks great! Thinking out longer term how many columns do you keep as time goes on? I can see that getting unwieldily.

**Reply by xfiderek** [reply]
Good point, we can hash-out the details of the format. Is there anything else we should discuss? e.g. where do we publish it, how do we maintain it. Anything else to consider?

**Comment by asimonov** [comment]
Great idea
